bigbluebutton-html5/imports/ui/stylesheets/variables/general.scss

```diff
@@ -13,3 +13,7 @@ $lg-padding-y: .5rem;
 
 $jumbo-padding-x: 3.025rem;
 $jumbo-padding-y: 1.25rem;
+
+$icon-top-offset: -.4em;
+$icon-left-offset: -.1em;
+$square-side-length: 1.56rem;
```

Comment:
Why those variables are in the general section?

Reply:
Wasn't sure if I was going to need to use them again for other elements. I'll move them back to the `chat-dropdown/styles.scss` where they are currently used.